linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/ansible/shared.yml

```diff
@@ -4,6 +4,9 @@
 # complexity = low
 # disruption = low
 
+{{# products that are available also in a 32 bits form #}}
+{{% if "rhel" not in product and product != "fedora" %}}
```

Comment:
Late to the party, but perhaps somebody notices: Why not to introduce an explanatory variable `product_has_32bit_form`? It's better to to pair products with properties, and then to work with those properties rather than with lists of products, mentioning the property as a comment.